src/main/java/core/basesyntax/UserService.java

```diff
@@ -3,9 +3,18 @@
 public class UserService {
     public void registerUser(User user) {
 
+        PasswordValidator passwordValidator = new PasswordValidator();
+
```

Comment:
remove all empty lines in the method
```suggestion
        PasswordValidator passwordValidator = new PasswordValidator();
```

Reply:
All empty lines in this method are removed)